torch/_dynamo/variables/nn_module.py

```diff
@@ -373,12 +373,12 @@ def generic_call_method_helper(name):
                 **options,
             )
 
-        if name == "_call_impl":
+        if name == "__call__":
```

Comment:
Can you have one example can trigger this call after this change? This was only used for explicit calling ```__call__``` in a forward function per my knowledge.

Reply:
After chatting offline, i think you're right, this looks wrong.

Also, _call_impl handling below looks wrong: it's not that it maps to forward, becuase it doesn't.  but it 'allows in graph' a whole call-impl including its hooks, which may not be safe.